### PR TITLE
THREESCALE-1699: Broken links in the admin portal

### PR DESCRIPTION
**Pull request by lvillen**
**What this PR does / why we need it**:

Updates broken links in the admin portal and rename "Follow the Quickstart guide" in Developer Portal > Content

**Which issue(s) this PR fixes** 

[THREESCALE-1699](https://issues.redhat.com/browse/THREESCALE-1699)

**Verification steps** 

1. Go to Products > Analytics > Response Codes and confirm link for "Response Codes Tracking" works
2. Go to Audience > Developer Portal > Content
3. Confirm that in the Quick Links section there's an item "Creating the Developer Portal" and that the link works.

**Response Codes**
Captura de pantalla 2023-07-20 a las 17 06 45

**Quick Links**
Captura de pantalla 2023-07-20 a las 17 06 27

### CODE REVIEW

**Review by jlledom — APPROVED** [review]
Tried locally, links work fine.

File: app/views/provider/admin/cms/templates/_quick_links.html.erb [hunk 1]
```diff
@@ -14,8 +14,8 @@
     <li>
       <p>
         <%= icon('question-circle fa-li') %>
-        Follow the
-        <%= link_to 'Quickstart guide', I18n.t('docs.hero_api_portal', docs_base_url: docs_base_url) %>
+        Creating the
+        <%= link_to 'Developer Portal', I18n.t('docs.hero_api_portal', docs_base_url: docs_base_url) %>
```
Comment (mayorova) [inline app/views/provider/admin/cms/templates/_quick_links.html.erb:18]:
```suggestion
        <%= link_to 'Creating the Developer Portal', I18n.t('docs.hero_api_portal', docs_base_url: docs_base_url) %>
```

File: app/views/provider/admin/cms/templates/_quick_links.html.erb [hunk 2]
```diff
@@ -14,8 +14,8 @@
     <li>
       <p>
         <%= icon('question-circle fa-li') %>
-        Follow the
-        <%= link_to 'Quickstart guide', I18n.t('docs.hero_api_portal', docs_base_url: docs_base_url) %>
+        Creating the
```
Comment (mayorova) [inline app/views/provider/admin/cms/templates/_quick_links.html.erb:17]:
```suggestion
        Check out documentation on
```

or whatever similar text so it's clear what the link is about

File: app/views/provider/admin/cms/templates/_quick_links.html.erb (hunk 1 above)
Comment (jlledom) [reply app/views/provider/admin/cms/templates/_quick_links.html.erb:18]: So `Creating the Creating the Developer Portal`?

File: app/views/provider/admin/cms/templates/_quick_links.html.erb (hunk 2 above)
Comment (jlledom) [reply app/views/provider/admin/cms/templates/_quick_links.html.erb:17]: Ah... I understand...

File: config/locales/en.yml [hunk 3]
```diff
@@ -540,7 +540,7 @@ en:
         documentation_hosted_url: '%{docs_base_url}/deployment-options/apicast-hosted'
         documentation_self_managed_url: '%{docs_base_url}/deployment-options/apicast-self-managed-nginx-config-files'
         documentation_oauth_url: '%{docs_base_url}/deployment-options/apicast-oauth-before-may-2017'
-        documentation_response_codes_tracking_url: '%{docs_base_url}/admin_portal_guide/response-codes-tracking'
+        documentation_response_codes_tracking_url: '%{docs_base_url}/admin_portal_guide/setting-up-and-evaluating-the-threescale-response-codes-log-for-your-api_analytics-for-threescale-apis'
```
Comment (mayorova) [inline config/locales/en.yml:543]: I think we need to change the text where this link is referenced also.

It's now
```
Our HowTo tutorial on [Response Codes Tracking](https://access.redhat.com/documentation/en-us/red_hat_3scale/2-saas/html/admin_portal_guide/response-codes-tracking) on the 3scale support portal
```
I'd change it to something like:
```
Documentation on [response codes logging](https://access.redhat.com/documentation/en-us/red_hat_3scale/2-saas/html/admin_portal_guide/setting-up-and-evaluating-the-threescale-response-codes-log-for-your-api_analytics-for-threescale-apis) on the Red Hat Customer Portal.
```

File: app/views/provider/admin/cms/templates/_quick_links.html.erb (hunk 1 above)
Comment (dfennessy) [reply app/views/provider/admin/cms/templates/_quick_links.html.erb:18]: Suggestion: 

Something like: `Follow the documentation on <link>Creating the Developer Portal</link>`

File: config/locales/en.yml (hunk 3 above)
Comment (dfennessy) [reply config/locales/en.yml:543]: Preferably we would remove the double occurrence of the word "on". So I suggest this:

`Documentation about response code logging on the Red Hat Customer Portal.`

**Review by dfennessy — APPROVED** [review]
LGTM!

File: app/views/application/_no_log_requests.html.slim [hunk 4]
```diff
@@ -14,6 +14,6 @@ ul.StatsMessages-list
   li The 3scale
     #{link_to('API ActiveDocs documentation', provider_admin_api_docs_path(anchor: '/service_management_api'), target:'_blank')}
     for our Service Management API methods where the logs with response codes can be sent.
-  li Our HowTo tutorial on logs and
-    #{link_to('Response Codes Tracking', I18n.t('api.integrations.proxy.documentation_response_codes_tracking_url'), target:'_blank')}
-    on the 3scale support portal
+  li Follow the documentation about
```
Comment (mayorova) [inline app/views/application/_no_log_requests.html.slim:17]: Actually.... I can't find where this view is included. Maybe we should delete it?

Comment (lvillen) [reply app/views/application/_no_log_requests.html.slim]: Yes, I'd say we can since nothing is rendering that partial, nice catch!

**Review by mayorova — APPROVED** [review]

**Pull request merged**